[Workers] Wrangler 1 -> 2 routes behaviour change (#5482)
cloudflare/cloudflare-docs: content/workers/wrangler/migration/deprecations.md

## Other deprecated behaviour

> - Wrangler defined routes will no-longer be added alongside routes added in the dashboard if you have a `route` or `routes` key in your `wrangler.toml`. It will instead only keep routes defined in `wrangler.toml`. To manage routes via the dashboard only, remove any `route`/`routes` keys from your `wrangler.toml` and add `workers_dev = false`.

Reviewer: ```suggestion
- Wrangler defined routes will not be added alongside routes configured in the Cloudflare dashboard if you have a `route` or `routes` key in your `wrangler.toml`. It will instead only keep routes defined in `wrangler.toml`. To manage routes via the Cloudflare dashboard only, remove any `route`/`routes` keys from your `wrangler.toml` and add `workers_dev = false`.
```

## Other deprecated behaviour

> - Wrangler defined routes will not be added alongside routes configured in the Cloudflare dashboard if you have a `route` or `routes` key in your `wrangler.toml`. It will instead only keep routes defined in `wrangler.toml`. To manage routes via the Cloudflare dashboard only, remove any `route`/`routes` keys from your `wrangler.toml` and add `workers_dev = false`.

Reviewer: ```suggestion
- Cloudflare dashboard-defined routes will not be added alongside Wrangler-defined routes. Wrangler-defined routes are the`route` or `routes` key in your `wrangler.toml`. If both are defined, only routes defined in `wrangler.toml` will be valid. To manage routes via the Cloudflare dashboard only, remove any `route` and `routes` keys from and add `workers_dev = false` to your `wrangler.toml` file.
```
